docs/01-app/04-deep-dive/image-optimization.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: How Next.js optimizes images
+nav_title: Image Optimization
+description: Learn how Next.js Image Optimization works.
+related:
+  title: API Reference
+  description: See the API Reference for the full feature set of Next.js Image
+  links:
+    - app/api-reference/components/image
+---
+
+Next.js provides automatic image optimization to improve the initial page load and user experience.
+
+In this deep dive, you'll learn how image optimization works in Next.js.
+
+## The Image Component
+
+The Next.js `<Image />` component extends the HTML `<img>` element with built-in best practices for optimizing images. It takes an unoptimized image and produces optimized outputs.
+
+```jsx filename="app/page.js"
+import Image from 'next/image'
+
+export default function Page() {
+  return <Image src="/profile.png" alt="Picture of the author" />
+}
+```
+
+> See the [Image Component API](/docs/app/api-reference/components/image) reference for all available props.
+
+### Image Sizing
+
+One of the most common image-related performance issues is [layout shift](https://web.dev/articles/cls). This happens when an image pushes other elements around the page as it loads in. It’s a UX problem that it has its own Core Web Vital: [Cumulative Layout Shift](https://web.dev/cls/). Layout shift can be prevented by defining the image dimensions. This allows the browser to reserve enough space for the image before it loads.
+
+The `<Image />` component is designed in a way that cannot contribute to layout shift, and **must** be sized in one of three ways:
+
+1. Automatically, using a local image as a static import.
+2. Manually, by including [`width`](/docs/app/api-reference/components/image#width-and-height) and [`height`](/docs/app/api-reference/components/image#width-and-height) props.
+3. Implicitly, by using [`fill`](/docs/app/api-reference/components/image#fill) to cause the image to expand to fill its parent element.
+
+### Supported image formats
+
+The built-in loader supports common web-friendly formats:
+
+- `jpeg`
+- `png`
+- `webp`
+- `avif`
+- `tiff`
```

Comment:
What are the supported vs. unsupported image formats?

Reply:
The `images.formats` config is how you configure the output type